pkg/puppet/puppet.go

```diff
@@ -233,7 +249,7 @@ func kubernetesInstancePoolConfig(conf *clusterv1alpha1.InstancePoolKubernetes,
 	return
 }
 
-func contentClusterConfig(cluster interfaces.Cluster) ([]string, error) {
+func contentClusterConfig(cluster interfaces.Cluster, packerBuild bool) ([]string, error) {
```

Comment:
Can we have packerBuild as part of the puppet struct and have this as a member func, iinstead of passing packerBuild around?